tests/integrated/test-coordinates-initialization/runtest

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+make
+# Kill the test after 3 seconds, if it hasn't already exited
+mpirun -n 4 timeout 3s ./test-coordinates-initialization
```

Comment:
Should we `mpirun` the `timeout` or `timeout` the `mpirun` (is there any difference, e.g. in the error message)?

As a sidenote openmpi provides `--timeout` option to mpirun and `mpich` provides `MPIEXEC_TIMEOUT` environment variable, unfortunately the standard doesn't mandate anything similar.

Reply:
I would suggest `timeout 3s mpirun ...`

Reply:
I found that `timeout 3s mpirun...` left processes running when timeout tried to stop the run, whereas `mpirun timeout...` finished cleanly but with non-zero exit code.